install.sh

```diff
@@ -357,13 +357,19 @@ nvm_do_install() {
     # shellcheck disable=SC2016
     if ${BASH_OR_ZSH} && ! command grep -qc '$NVM_DIR/bash_completion' "$NVM_PROFILE"; then
       echo "=> Appending bash_completion source string to $NVM_PROFILE"
+      if [ -n "$ZSH_VERSION" ]; then
+        command printf "$ZSH_COMPLETION_STR" >> "$NVM_PROFILE"
+      fi
       command printf "$COMPLETION_STR" >> "$NVM_PROFILE"
     else
       echo "=> bash_completion source string already in ${NVM_PROFILE}"
     fi
   fi
   if ${BASH_OR_ZSH} && [ -z "${NVM_PROFILE-}" ] ; then
     echo "=> Please also append the following lines to the if you are using bash/zsh shell:"
+    if [ -n "$ZSH_VERSION" ]; then
+        command printf "${ZSH_COMPLETION_STR}"
```

Comment:
2-space indentation

Reply:
Removed in favor of using `bash_completion`